Require WindowMaterial:ComplexShade Emissivity to be >0

LipingWang opened the pull request:
Pull request overview
---------------------
 - Fixes #7698 
 - Change the minimum requirement of the emissivity (Front and Back ) to be greater than zero rather than zero in Energy+.idd.in.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

LipingWang committed:
Update Energy+.idd.in

https://github.com/NREL/EnergyPlus/issues/7698
Change the minimum requirement of the emissivity (Front and Back ) to be greater than zero rather than zero in Energy+.idd.in.

LipingWang committed:
WindowMaterial:ComplexShade Emissivity to be >0

issue #7708
Added the unit test for WindowMaterial:ComplexShade

LipingWang committed:
Merge branch 'develop' into Emissivity-in-WindowMaterial-ComplexShade

mitchute commented:
@LipingWang the documentation needs to be updated:
https://github.com/NREL/EnergyPlus/blob/7e756614672eacf6e7cc71267a015a67f7ba2073/doc/input-output-reference/src/overview/group-surface-construction-elements.tex#L2573

https://github.com/NREL/EnergyPlus/blob/7e756614672eacf6e7cc71267a015a67f7ba2073/doc/input-output-reference/src/overview/group-surface-construction-elements.tex#L2577

Once you have corrected that, please merge in develop on more time.

LipingWang committed:
Merge branch 'develop' into Emissivity-in-WindowMaterial-ComplexShade

LipingWang committed:
Update group-surface-construction-elements.tex

Updated the file "group-surface-construction-elements.tex" in the folder doc\input-output-reference\src\overview.

mitchute commented:
Not expecting any problems from the last VS build since it's been always been clean. Merging.
